core/components/molecules/table/table.js

```diff
@@ -173,6 +176,20 @@ Table.Cell = styled.td`
   width: ${props => props.column.width || 'auto'};
 `
 
+Table.EmptyState = ({ rows, children }) => {
+  if (rows.length > 0 || !children) return null
+
+  const TableEmptyState = styled.div`
+    padding: ${spacing.small};
+    background-color: rgb(250, 250, 250);
+    border-radius: 5px;
+    text-align: center;
+    margin-top: ${spacing.xsmall};
+  `
```

Comment:
Let's add a `color: $colors.text.default` rule so the typography uses the default text color